docs/core/migration/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: .NET Core project.json to csproj migration | Microsoft Docs
+description: .NET Core project.json to csproj migration
+keywords: .NET, .NET Core, .NET Core migration
+author: blackdwarf
+ms.author: mairaw
+ms.date: 02/17/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 1feadf3d-3cfc-41dd-abb5-a4fc303a7b53
+---
+
+# Migrating .NET Core projects
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+This document will cover migration scenarios for .NET Core projects and will go over the following three migration scenarios:
+
+1. Migration from a valid latest schema of *project.json* to *csproj*.
+2. Migration from DNX to a valid Preview 2 project.json.
+3. Migration from RC3 and previous .NET Core csproj projects to the final format .
+
+## Migration from project.json to csproj
```

Comment:
Yes, I was thinking about adding that here too. It'd fix this issue (https://github.com/dotnet/docs/issues/1543)